cdap-app-templates/cdap-dq/src/main/java/co/cask/cdap/dq/DataQualityApp.java

```diff
@@ -88,45 +84,8 @@
     private String datasetName;
     private Map<String, Set<String>> fieldAggregations;
 
-    public ConfigClass() {
```

Comment:
while we're at it, lets change the name of this to DQConfig or something like that.